plugins/sdm/accessbot.py

```diff
@@ -34,6 +39,14 @@ def callback_message(msg):
         ErrBot.callback_message(bot, msg)
     return callback_message
 
+def get_send_simple_reply(bot):
+    def send_simple_reply(msg, text, private=False, threaded=False):
```

Comment:
Validate if we can better change in errbot where the exceptions are being handled. Specifically when it returns: `MSG_ERROR_OCCURRED`

Reply:
Unfortunately the exceptions are captured and handled internally on Errbot. Take a look [here](https://github.com/errbotio/errbot/blob/master/errbot/core.py#L598).